robo_arm:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint_body_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_body
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint_shoulder_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_shoulder
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint_elbow_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_elbow
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint_finger1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_finger1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint_finger2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_finger2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint_finger3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_finger3
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint_finger4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_finger4
    pid: {p: 100.0, i: 0.01, d: 10.0}
